cmd/podman/common.go

```diff
@@ -28,6 +28,10 @@ var (
 		Name:  "latest, l",
 		Usage: "act on the latest pod podman is aware of",
 	}
+	WorkDirFlag = cli.StringFlag{
+		Name:  "workdir, w",
```

Comment:
it should be added to completions/bash/podman as well

Reply:
Added.